src/Factory/HalConfigFactory.php

```
@@ -5,8 +5,11 @@
 namespace Laminas\ApiTools\Hal\Factory;
 
 use ArrayAccess;
+// phpcs:disable WebimpressCodingStandard.PHP.CorrectClassNameCase.Invalid
 use Interop\Container\ContainerInterface;
 
+// phpcs:enable WebimpressCodingStandard.PHP.CorrectClassNameCase.Invalid
+
```

You can use `phpcs:ignore` to ignore an error on a single line, instead of using disable/enable pairs:

```php
// phpcs:ignore WebimpressCodingStandard.PHP.CorrectClassNameCase.Invalid
use Interop\Container\ContainerInterface;
```

---

Hi, I used enable/disabled to prevent ignoring other class names on the file, but I will apply the required change.

---

[From the phpcs wiki](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file):

> You can also ignore a single line using the phpcs:ignore comment. If placed on a line by itself, this comment will ignore the line that the comment is on and the following line. It is typically used like this:
> 
> ```php
> // phpcs:ignore
> $foo = [1,2,3];
> bar($foo, false);
> ```
> 
> Or if the comment is placed on a line that also contains code, only that line will be ignored:
>
> ```php
> $foo = [1,2,3]; // phpcs:ignore
> bar($foo, false);
> ```

The `phpcs:disable` usage is the one that will ignore everything following it, until a `phpcs:enable` statement is provided.